nvchecker/gen-config.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+cd "${0%/*}/.."
+ROOT="$(pwd)"
+
+CFG=nvchecker/archlinux-proaudio.toml
+VER=nvchecker/old_ver.json
+cat > $CFG << __EOF__
+[__config__]
+oldver = "old_ver.json"
+newver = "new_ver.json"
+keyfile = "keyfile.toml"
+
+__EOF__
+
+VER_TEMP="$(mktemp)"
+for f in packages/*/.nvchecker.toml; do
+  cat $f >> $CFG
+  echo "" >> $CFG
+  pkgdir=$(dirname $f)
+  pkgbase=$(basename $pkgdir)
+  pkgver=$(. $pkgdir/PKGBUILD; echo $pkgver)
```

Comment:
I'm not sure how I feel about this. Since this is executed on the repo server directly, this seems like an avenue too easy to exploit for executing arbitrary stuff. Aren't there any tools, which can parse the information from PKGBUILD files more safely?

Reply:
Good point, my reasoning was basically that the server pulls only from `master` which we can (and should) expect to trust, otherwise the attack could as well just happen inside one of these nvchecker scripts itself.

There are probably 10 more ways to do the same thing, but one idea could be:
1. CI generates nvchecker configs on push to master
2. configs get uploaded to website
3. cronjob (either on the existing server or from a [CI cronjob](https://docs.drone.io/cron/) pulls configs from website to execute

regarding just parsing the PKGBUILD here, this should do:
```suggestion
  pkgver=$(sed -n 's/^pkgver=//p' $pkgdir/PKGBUILD)
```

Reply:
That wouldn't handle quotes correctly, wouldn't it? They would become part of the value.

Reply:
27e83b4f24bdcb8edb0bf47c1835ee638c1e1ea9

Reply:
fixed in 62fb3d9d1a0ee590946210f57523dc07a8499e8f

Reply:
ok